app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -538,7 +548,7 @@ watchEffect(() => {
 
   // If data is truncated, we cannot rely on sorting/filtering so will disable.
   options.defaultColDef.filter = !isTruncated.value
-  options.defaultColDef.sortable = !isTruncated.value
+  // options.defaultColDef.sortable = !isTruncated.value
```

Comment:
Let’s not keep the commented code without explanation. We can either remove it, or add a comment explaining why it is commented and when it will be uncommented.

Reply:
```suggestion
```